packages/menu/src/MenuItem.ts

```diff
@@ -83,6 +90,41 @@ export class MenuItem extends ActionButton {
         return content;
     }
 
+    protected render(): TemplateResult {
+        return this.href && this.href.length > 0
+            ? this.renderAnchor({
+                  id: 'button',
+                  className: 'button',
+                  anchorContent: this.buttonContent,
+              })
+            : html`
+                  <div
+                      id="button"
+                      class="button"
+                      aria-label=${ifDefined(this.label)}
+                  >
+                      ${this.buttonContent}
+                  </div>
+              `;
+    }
```

Comment:
Hadn't realized this extended from ActionButton -- neat. Whenever something like a core render method is overridden I try to think about maintainability. Since the only difference is the tag name, is that something you could parameterize?

Reply:
We could make the source render method more composable so that we only have to override part of it:
```
protected render(): TemplateResult {
    return this.isAnchor ?  this.renderAnchor : this.renderButton;
}
protected get renderAnchor() {}
protected get renderButton() {}
```
That's probably nicer system wide, I'll take  swing at it.